src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj

```diff
@@ -12,6 +12,11 @@
         <Content Condition=" '$(BlazorMode)' == 'BlazorWebAssembly' " Remove="Pages\_Host.cshtml" />
         <Content Condition=" '$(BlazorMode)' == 'BlazorWebAssembly' " Remove="Pages\_Layout.cshtml" />
         <Content Remove="compilerconfig.json" />
+        <Content Include="Styles\abstracts\_fonts.scss" />
```

Comment:
You've to make sure that you understand every change that you've made and all changes are required in your opinion (":
Please revert changes in csproj file as these are not required at all
Note that I understand that sometimes visual studio make these changes

Reply:
yeah, I see it a few minutes ago and I'm  fixing it

Reply:
it's done now